frontend/src/main-components/MyFoldersContent.js

```diff
@@ -9,11 +10,54 @@ const MyFoldersContent = (props) => {
     border-radius: 1rem;
     background-color: white;
     margin: 1%;
+    flex-direction: column;
   `;
+  const folderStyle = {
+    display: "flex",
+    flexFlow: "row wrap",
+    justifyContent: "space-around",
+    alignItems: "start",
+  };
+
+  let folders;
+
+  //Fetch data from the UserFolders servlets to get the user folders
+  //Currently there is no way to get the userid of the user
```

Comment:
What do you mean by there is no way to get the user id? Do you mean the userKey?

Reply:
Yeah.